modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/EditItem.js

```diff
@@ -194,53 +195,69 @@ function EditItem() {
 				</div>
 
 				<div>
-					<div className="mini-cart-prices p-4">
-						<PriceRow
-							priceName={Liferay.Language.get('list-price')}
-						>
-							{hasPromoPrice || hasDiscount ? (
-								<span className="price-line-through">
-									{price.priceFormatted}
-								</span>
-							) : (
-								<span>{price.priceFormatted}</span>
-							)}
-						</PriceRow>
-
-						{hasPromoPrice ? (
+					{!priceOnApplication && (
+						<div className="mini-cart-prices p-4">
 							<PriceRow
-								priceName={Liferay.Language.get('promo-price')}
+								priceName={Liferay.Language.get('list-price')}
 							>
-								{hasDiscount ? (
+								{hasPromoPrice || hasDiscount ? (
 									<span className="price-line-through">
-										{price.promoPriceFormatted}
+										{price.priceFormatted}
 									</span>
 								) : (
-									<span>{price.promoPriceFormatted}</span>
+									<span>{price.priceFormatted}</span>
 								)}
 							</PriceRow>
-						) : null}
 
-						{hasDiscount ? (
+							{hasPromoPrice ? (
+								<PriceRow
+									priceName={Liferay.Language.get(
+										'promo-price'
+									)}
+								>
+									{hasDiscount ? (
+										<span className="price-line-through">
+											{price.promoPriceFormatted}
+										</span>
+									) : (
+										<span>{price.promoPriceFormatted}</span>
+									)}
+								</PriceRow>
+							) : null}
+
+							{hasDiscount ? (
+								<PriceRow
+									priceName={Liferay.Language.get('discount')}
+								>
+									<span className="price-discount">
+										{`-${price.discountPercentage}%`}
+									</span>
+								</PriceRow>
+							) : null}
+
 							<PriceRow
-								priceName={Liferay.Language.get('discount')}
+								priceName={Liferay.Language.get(
+									'price-as-configured'
+								)}
 							>
-								<span className="price-discount">
-									{`-${price.discountPercentage}%`}
+								<span className="text-7">
+									{price.finalPriceFormatted}
 								</span>
 							</PriceRow>
-						) : null}
-
-						<PriceRow
-							priceName={Liferay.Language.get(
-								'price-as-configured'
-							)}
-						>
-							<span className="text-7">
-								{price.finalPriceFormatted}
-							</span>
-						</PriceRow>
-					</div>
+						</div>
+					)}
+
+					{priceOnApplication && (
+						<div className="mini-cart-prices p-4">
+							<PriceRow>
```

Comment:
Is it correct to don't have the "label" on the left if it's price on application?